File: libraries/AP_HAL_ChibiOS/hwdef/BRGR/scripts/helloworld.lua
-- hello.lua
-- Simple "Hello World" script for ArduPilot Plane

function update()
    -- Send a message to the GCS console
    gcs:send_text(6, "Hello, world from Lua on Plane!")
    
    -- Run this function again in 1000 ms (1 second)
    return update, 1000
end

-- Start the update loop
return update, 1000

File: modules/waf/playground/gtest/src/HelloWorld.cpp

#include "HelloWorld.h"
HelloWorld::HelloWorld():m_message("Hello World")
{
}

HelloWorld::HelloWorld(const std::string & msg):m_message(msg)
{}

std::string HelloWorld::message() const
{
  return m_message;
}

void HelloWorld::setMessage(const std::string & msg)
{
  m_message = msg;
}
